.github/workflows/component-builder.yml

```diff
@@ -1,62 +1,134 @@
 name: "Component Builder Check"
 on:
-  pull_request:
+  # Wait for event on pull_request_target so that this action
+  # can access repo secrets: QUAY_USER & QUAY_TOKEN
```

Comment:
why not use the vars from bitwarden?

Reply:
You have BITWARDEN_ORGANIZATION_ID and BITWARDEN_ACCESS_TOKEN, but I don't think it's smart to use them, in case of some security leaks/issues I prefer to only expose the GitHub token and not Bitwarden creds